tests/functional/test_package.py

```diff
@@ -162,9 +163,17 @@ def osutils():
 
 
 @pytest.fixture
-def pip_runner():
+def empty_env_osutils():
```

Comment:
Does it makes sense to subclass from `OSUtils` for the `EmptyEnv` class?

Reply:
It won't really help or hurt anything in terms of the actual code, but it may make it clearer to someone reading the tests where this came from and what (partial) interface its trying to replace so I will update that.